infrastructure/rdfdb/cdk/docker/scripts/setup.sh

```diff
@@ -4,14 +4,17 @@ set -e
 echo 'org.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource' >> /usr/local/tomcat/conf/catalina.properties
 echo "RDF4J_DATA_DIR is set to: ${RDF4J_DATA_DIR}"
 
-/usr/local/tomcat/setup_web_auth.sh
+# /usr/local/tomcat/setup_web_auth.sh
```

Comment:
I'm surprised this wasn't erroring out before?

Reply:
Good catch, I think I committed too much